Makefile

```diff
@@ -96,17 +97,21 @@ DISTROS =           \
 	ubuntu-18.04    \
 
 # Template for running tests inside a Linux distro container
-DRUN = docker run                                 \
-	--env COLUMNS=${COLUMNS}                      \
-	--init                                        \
-	--tty                                         \
-	--mount type=bind,source=${PWD},target=/ebash \
-	--mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
-	--network host                                \
-	--privileged                                  \
-	--rm                                          \
-	--workdir /ebash                              \
+DRUN = bin/ebash docker_run          \
+	--envlist "${ENVLIST}"           \
+	--nested                         \
+	--copy-to-volume "${PWD}:/ebash" \
+	--                               \
```

Comment:
is this empty option needed?

Reply:
Yes, the `--` tells `docker_run` to stop parsing options. Everything after the empty `--` is passed through direct to docker rather than being handled by `docker_run`. This is a common GNU idiom.

Reply:
Thanks for the explaination Marshall.